### PR TITLE
DOC: Added health check and team management

### PR DESCRIPTION
**Pull request by writinwaters**
### Type of change

- [x] Documentation Update

### CODE REVIEW

File: docs/guides/manage_team_members.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 4
+slug: /manage_team_members
+---
+
+# Manage team members
+
+Invite or remove team members, join or leave a team.
+
+By default, each RAGFlow user is assigned a single team named after their name. RAGFlow allows you to invite an unlimited number of RAGFlow users to your team. Your team members can help you:
```
Comment (JinHai-CN) [inline docs/guides/manage_team_members.md]: Remove: an unlimited number of

File: docs/guides/manage_team_members.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+![team](https://github.com/user-attachments/assets/0eac2503-26bc-4568-b3f2-bcd84069a07a)
+
+_On the **Team** page, you can view the information about members of your team and the teams you have joined._
+
+## Invite team members
+
+You are, by default, the owner of your own team and the only person permitted to invite users to join your team or remove team members.
+
+:::tip NOTE
+Only RAGFlow users can be invited for now.
```
Comment (JinHai-CN) [inline docs/guides/manage_team_members.md]: Remove the tip

File: docs/guides/run_health_check.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- **Elasticsearch** (default) or [Infinity](https://github.com/infiniflow/infinity) as the document search engine
+- **MySQL**
+- **Redis**
+- **MinIO** for object storage
+
+If an exception or error occurs related to any of the above services, such as `Exception: Can't connect to ES cluster`, refer to this document to check their health status.
+
+## Command line
+
+If you installed RAGFlow using Docker, run this command to list all Docker containers running on your host machine and their health status:
```
Comment (JinHai-CN) [inline docs/guides/run_health_check.md]: installed->deployed

File: docs/guides/run_health_check.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+slug: /run_health_check
+---
+
+# Run health check on RAGFlow's base services
+
+Double check the health status of RAGFlow's base services.
+
+The operation of RAGFlow depends on four base services:
+
+- **Elasticsearch** (default) or [Infinity](https://github.com/infiniflow/infinity) as the document search engine
```
Comment (yuzhichang) [inline docs/guides/run_health_check.md]: `document search engine` -> `document engine`

File: docs/guides/run_health_check.md (hunk 3 above)
Comment (JinHai-CN) [reply docs/guides/run_health_check.md]: Remove `command line` chapter

File: docs/guides/run_health_check.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 7
+slug: /run_health_check
+---
+
+# Run health check on RAGFlow's base services
```
Comment (JinHai-CN) [inline docs/guides/run_health_check.md]: base services -> dependencies

**Pull request merged**